packages/ui-components/src/components/DataGrid.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DataGrid as MuiDataGrid } from '@mui/x-data-grid';
+
+export const DataGrid = ({ columns, rows }) => {
+  return (
+    <div style={{ height: '100%', width: '100%' }}>
+      <div style={{ display: 'flex', height: '100%' }}>
+        <div style={{ flexGrow: 1 }}>
+          <MuiDataGrid columns={columns} rows={rows} density="compact" disableSelectionOnClick />
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
I think for consistency's sake, instead of having inline styling, making `styled-components` for each one of these divs in this file would make sense (you can just make them in this same file and put above `DataGrid`).

Reply:
yes, thanks!

Reply:
Also what are these 3 divs meant to be doing? I wonder it could be done with 2 or even one div?